examples/p5js/Handpose/Handpose_Image/sketch.js

```diff
@@ -0,0 +1,52 @@
+let handpose;
+let predictions = [];
+let img;
+
+function setup() {
+  // Create a canvas that's at least the size of the image.
+  createCanvas(400, 350);
+
+  // create an image using the p5 dom library
+  // call modelReady() when it is loaded
+  img = createImg("data/hand.jpg", imageReady);
```

Comment:
I think this could be `loadImage()`

And we might actually put the image and model loading into a `preload()` like:

```js
function preload(){
img = loadImage(....);
handpose = ml5.handpose();
}
```


```